biodata-models/src/main/java/org/opencb/biodata/models/variant/Variant.java

```diff
@@ -332,25 +332,33 @@ public void addEffect(String allele, VariantEffect ct) {
         return file.getSampleNames();
     }
 
+    /**
+     * see http://www.ensembl.org/info/docs/tools/vep/vep_formats.html
+     */
     public void transformToEnsemblFormat() {
```

Comment:
This transformation will make the already existing Variant object non-compliant with the rules assumed by the OpenCB tools. It would be safer to return a new Variant object with the new fields.